src/transformers/modeling_albert.py

```diff
@@ -216,7 +216,6 @@ def __init__(self, config):
         # position_ids (1, len position emb) is contiguous in memory and exported when serialized
         self.register_buffer("position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)))
 
-    # Copied from transformers.modeling_bert.BertEmbeddings.forward
```

Comment:
This new feature would render BERT and ALBERT embedding different, which I think is fine